apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1333,14 +1349,11 @@ const styles = {
     fontWeight: 100,
   },
   listItems: {
-    margin: '0 10px 0 0',
+    margin: '10px',
```

Comment:
Nit: I think this can just be `margin: 10` now (definitely not worth a rebuild if this is the only change)

Reply:
Good catch- updated!